.coveragerc

```diff
@@ -1,6 +1,9 @@
 [report]
 # Regexes for lines to exclude from consideration
 exclude_lines =
+    def __repr__
+    os.remove
```

Comment:
Why this line?

Reply:
There are 2 `os.remove` operations in the core files. They can be reached only when KeyboardInterrupt occurs.